README.md

```diff
@@ -41,7 +41,8 @@ module.exports = {
     options: {
       path: "logs/", // path to write files to
       size: "10M", // max file size
-      rotate: 5 // keep 5 rotated logs
+      rotate: 5, // keep 5 rotated logs
+      isJson: true // console logs will be in json format if set true else string.
```

Comment:
Since one of the primary features and uses for this module is for JSON formatting, I don't want it to seem like someone should need to set `isJson: true`. Instead, I suggest the following
```suggestion
      // isJson: false - JSON formatting will be disabled
```